### ISSUE TITLE
Nelze připsat alergie ke členovi z databáze

### ISSUE THREAD

**Issue by hla-bla** [bosancz/interni-sekce#52]
Ahoj,

při úpravě člena v databázi nelze připsat jeho alergie. Neexistuje na to kolonko v tom upravovacím módu. V náhledu toho člena to také nelze natvrdo vepsat. Poznámka k verzi 3.0.9

Fíla

**Comment by SmallhillCZ** [comment]
@hla-bla jo, to jsme tam zatím udělali jen jako kartičku, ale jinak nenaimpelmentovali editaci a uložení do databáze. ale tak je super, že by to očividně mělo využití, takže pro nás signál, abychom to dodělali.

@JakubRejzek tak co, troufal by sis na to? Zatím to není naimplementované ani na backendu, ale můžeme si to nějak rozdělit.
